HHS/Head-Start-TTADP
# Fix grammar

## Description of change
Fixes the grammatical inconsistencies flagged in TTAHUB-1190


## How to test
Verify the e2e tests pass.
On the review page, verify the grammar is correct as described in 1190

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1190


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions


### Production Deploy


### After merge/deploy
